components/workspacekit/cmd/rings.go

```diff
@@ -546,6 +546,23 @@ var ring1Cmd = &cobra.Command{
 			}
 		}()
 
+		go func() {
```

Comment:
Just out of curiosity: why do this in workspacekit rather than supervisor? The latter has two benefits: 
1. we could forward the output to a terminal if we wanted to
2. it integrates nicer with full-workspace backup for which we'd need to move this to supervisor

Reply:
That said, I don't think that's a blocker and I'd be ok with having this go in the way it is